Gems/Atom/Feature/Common/Code/Source/SkinnedMesh/SkinnedMeshOutputStreamManager.cpp

```diff
@@ -80,7 +82,7 @@ namespace AZ
         AZ_CVAR(
             int,
             r_skinnedMeshInstanceMemoryPoolSize,
-            256,
+            AZ_TRAIT_DEFAULT_SKINNING_MEMORY_SIZE,
```

Comment:
Why do we need to PAL-ify if we are already using buffer size limit from the device itself?

Reply:
Because that sets a more suitable default memory to use for skinning on each platform to the `r_skinnedMeshInstanceMemoryPoolSize` cvar, otherwise on mobiles it will always print the warning that it's trying to use too much memory for skinning buffers and that can lead to think there is something wrong to be addressed.